mypy/test/testapi.py

```diff
@@ -13,12 +17,18 @@ def setUp(self) -> None:
         self.sys_stderr = sys.stderr
         sys.stdout = self.stdout = StringIO()
         sys.stderr = self.stderr = StringIO()
+        with NamedTemporaryFile(delete=False) as tmp:
+            tmp.write(b"x: int = 5\n")
+        self.tmp_path = Path(tmp.name)
+        self.tmp_cache_dir = Path(mkdtemp())
```

Comment:
This will get run before every test. Should this get moved into `test_default_encoding_warnings`?